src/predicate.ts

```diff
@@ -139,12 +141,20 @@ export interface FieldOneOfPredicate extends FieldPredicateBase {
   oneOf: string[] | number[] | boolean[] | DateTime[];
 }
 
+export interface FieldValidPredicate extends FieldPredicateBase {
+  valid: true;
```

Comment:
This should support false too.  Users may want to analyze the pattern of data that have some fields missing.

Reply:
You can always use `not`. I'd rather support fewer ways to do the same thing for now. We can change it later.

Reply:
I disagree.  Users wouldn't expect that it can't be `false`, so better fix it now since it's very easy and provides a way nicer syntax that nesting it under "not".